api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -23,6 +23,21 @@ const (
 	ActionRelocate = DRAction("Relocate")
 )
 
+// DRPCType is the type of the DRPC based on the application handle
+type DRPCType string
+
+// These are the valid values for DRPCType
+const (
+	// OCMManaged is the DRPC type for the applications that are managed by OCM
+	OCMManaged = DRPCType("OCM Managed")
```

Comment:
Remove OCM from names, we can just call these `Managed`, 'Discovered' (or maybe Unmanaged)